params/types/multigeth/multigethv0_chain_config_configurator.go

```diff
@@ -313,6 +313,8 @@ func (c *ChainConfig) SetEIP1052Transition(n *uint64) error {
 }
 
 func (c *ChainConfig) GetEIP1283Transition() *uint64 {
+	// This is special case handling for multigeth configuration.
+	// It makes the tests pass.
 	if c.ConstantinopleBlock != nil && c.PetersburgBlock != nil {
 		if c.ConstantinopleBlock.Cmp(c.PetersburgBlock) == 0 {
 			return nil
```

Comment:
This is not an ideal thing, and it differs from the `goethereum` implementation. This is due, in part, to core-geth's original inheritance of the multi-geth configuration format and need to support some old strange versions of the multi-geth config.

Reply:
I looked at generalizing the interaction of `EIP1283` and `EIP1283Disable` at the `Configurator.(Equivalent|Compatible)` level, but decided that was too involved for now.